gcp-serverless-endpoint/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "resource_name" {
+  type = string
+}
+
+variable "labels" {
+  type = any
+}
+
+variable "subdomain" {
```

Comment:
future facing thing in prep for supporting "service URLs" it would be nice if this module had an output that produced the `https` url this endpoint could be reached at. I also could see this being used in the artifact produced by bundles consuming this module. i.e. several serverless microservices each take in some endpoints of other services that could be represented as lines / connections on a massdriver canvas.